### PR TITLE
Assignment 6 by Shijia

### PR DESCRIPTION
**Pull request by shijialol**

### CODE REVIEW

File: Assignment6.Rmd [hunk 1]
```diff
@@ -23,18 +23,19 @@ library(rpart)
...
+M2 <- read.csv("MOOC2.csv", header = TRUE)
 
 ```
 
 #Decision tree
 ```{r}
 #Using the rpart package generate a classification tree predicting certified from the other variables in the M1 data frame. Which variables should you use?
 
-c.tree1 <- 
+c.tree1 <- rpart(certified ~ forum.posts + grade + assignment, method ="class", data=M1, control = rpart.control(minsplit = 1, minbucket = 1, cp=0.001))
```
Comment (charles-lang) [inline Assignment6.Rmd]: You may need to `scale()` your variables as `forum.posts` has many more increments and will be preferenced by the algorithm.

File: Assignment6.Rmd [hunk 2]
```diff
@@ -48,7 +49,7 @@ post(c.tree1, file = "tree1.ps", title = "MOOC") #This creates a pdf image of th
 #If we are worried about overfitting we can remove nodes form our tree using the prune() command, setting cp to the CP value from the table that corresponds to the number of nodes we want the tree to terminate at. Let's set it to two nodes.
 
 ```{r}
-c.tree2 <- prune(c.tree1, cp = )#Set cp to the level at which you want the tree to end
+c.tree2 <- prune(c.tree1, cp = 0.014545)#Set cp to the level at which you want the tree to end
```
Comment (charles-lang) [inline Assignment6.Rmd]: Good job, what was the outcome of pruning?

File: Assignment6.Rmd [hunk 3]
```diff
@@ -65,5 +66,10 @@ M2$predict2 <- predict(c.tree2, M2, type = "class")
 table(M2$certified, M2$predict1)
 
 table(M2$certified, M2$predict2)
+M2$error1 <- ifelse(M2$certified == M2$predict1, 1, 0)
+M2$error2 <- ifelse(M2$certified == M2$predict2, 1, 0)
+
+summary(M2$error1)
+summary(M2$error2)
```
Comment (charles-lang) [inline Assignment6.Rmd]: Do you have an explanation for this analysis?

**Comment by charles-lang** [comment]
Hi Shijia,

It just means that I have added comments to your pull request. You should be able to access them through your account, I usually close the pull request after I have commented on your assignment. Does that make sense?

Best,

Charles
Visiting Assistant Professor in Learning Analytics | Dept. of Human Development | 
Teachers College, Columbia University | www.tc.columbia.edu/human-development/

**Comment by sw3241** [comment]
Yes, It does, I will let her know:)

**Pull request closed without merging**